templates/employee_profile.html

```diff
@@ -1,93 +1,94 @@
 {% extends "base.html" %}
+{% load i18n %}
 
 {% block content %}
     <div class="container mt-5">
 
         <ul class="list-group">
             <li class="list-group-item d-flex">
                 <div class="mr-15">
-                    Username :
+                    {% trans 'Username :' %}
```

Comment:
```suggestion
                    {% trans 'Username' %} :
```
better to exclude symbols from words

Reply:
fixed, thank you.